src/events/mouse.js

```diff
@@ -11,6 +11,47 @@
 var p5 = require('../core/main');
 var constants = require('../core/constants');
 
+/**
+ *
+ * The variable movedX contains the vertical movement of the mouse since the last frame
+ * @property {Number} movedX
+ * @readOnly
+ * @example
+ * <div class="notest">
+ * <code>
+ * let x = 0;
+ * function draw() {
+ *   x = (x + movedX) % 100;
+ *   background(237, 34, 93);
+ *   fill(0);
+ *   rect(x, 50, 50, 50);
+ * }
+ * </code>
+ * </div>
+ *
+ *
+ */
+p5.prototype.movedX = 0;
+
+/**
+ * The variable movedY contains the horizontal movement of the mouse since the last frame
+ * @property {Number} movedY
+ * @readOnly
+ * @example
+ * <div class="notest">
+ * <code>
+ * let y = 0;
+ * function draw() {
+ *   y = (y + movedY) % 100;
```

Comment:
I really like the look of these two examples but the interaction behavior feels a bit glitchy. I wonder if there is a way to smooth the behavior a little bit without adding too much complexity to the code?

Reply:
Maybe a good chance to show off a swipe like interaction?

Reply:
also wondering if we can remove the `%` operator, I think this could be unfamiliar to many beginners

Reply:
how about an example where it always moves towards the center slowly and then moves according to the mouse movement oO ?

Reply:
```
let x = 0;
function setup(){
   rectMode(CENTER)
}
function draw() {
  if(x > 50){
     x--;
  } else if( x < 50 ){
     x++
  }
  x += floor(movedX);
  background(237, 34, 93);
  fill(0);
  rect(x, 50, 50, 50);
}
```

Reply:
this is how i mean slowly moves towards the center and reacts to mouse move

Reply:
working demo https://editor.p5js.org/jonathanheindl/present/cY13G4WIr

Reply:
This looks good! If you have time to change these examples right now, we are about to make the 0.9.0 release and this feature could make it in!

Reply:
ah sorry Im operatiing on eu time zone

Reply:
No sweat! It'll make it into 0.10.0